extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/DeviceHandler.java

```diff
@@ -283,15 +289,15 @@ private synchronized BridgeHandler getDssBridgeHandler() {
 
             if (handler instanceof BridgeHandler) {
                 dssBridgeHandler = (BridgeHandler) handler;
-                if (device == null) {
-                    dssBridgeHandler.registerDeviceStatusListener(this);
-                }
+                // if (device == null) {
```

Comment:
maybe its better to fully remove these lines instead of just commenting them out.